src/CMakeLists.txt

```diff
@@ -348,7 +340,10 @@ if(USE_3DMAPPER)
 endif()
 
 if(WIN32)
-    target_link_libraries(mudlet ws2_32
+    target_link_libraries(mudlet ws2_32)
+    set_target_properties(mudlet
+      PROPERTIES
+        WIN32_EXECUTABLE ON
```

Comment:
This seems new? We should double-check if everything works with this flag on, including the stdout console output we managed to make work on Windows.

Reply:
Ah I see we had it as `WIN32` before.

Reply:
Yes. This and the `add_executable` option are the same according to https://cmake.org/cmake/help/v3.3/command/add_executable.html